package.json

```diff
@@ -60,10 +52,18 @@
     "typescript": "^4.9.4",
     "vitest": "^0.28.5"
   },
-  "packageManager": "pnpm@7.9.5",
+  "preconstruct": {
+    "packages": [
+      "packages/*"
+    ]
+  },
+  "packageManager": "pnpm@8.10.2",
   "pnpm": {
     "patchedDependencies": {
       "html-render-webpack-plugin@3.0.1": "patches/html-render-webpack-plugin@3.0.1.patch"
     }
+  },
+  "volta": {
+    "node": "20.9.0"
```

Comment:
Can we keep the `pnpm` volta config, or does this cause an error if you don't enable have the volta envar set?

Reply:
We can, however it's prone to be forgotten (as was the case with this PR). Upgrading pnpm means you need to change `packageManager`, as well as the `volta.pnpm` keys.

`packageManager` is more of a de facto standard, which is why I added it.